code/modules/hippie/code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -0,0 +1,25 @@
+// -------------------------------------
+// Kitchen Smartfridge
+// -------------------------------------
+/obj/machinery/smartfridge/kitchen
+	name = "harvest storage"
+	desc = "Holds and preserves organically grown biomasses. Otherwise known as plants."
+	var/list/spawn_harvest = list(
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tomato = 3,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/vulgaris = 3,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/eggplant = 2,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/carrot = 2,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/corn = 2,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/apple = 2,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/wheat = 2)
+
+
+/obj/machinery/smartfridge/kitchen/New()
+	..()
+	for(var/typekey in spawn_harvest)
+		var/amount = spawn_harvest[typekey]
+		if(isnull(amount)) amount = 1
+		while(amount)
```

Comment:
Why?
There would not be no point listing items to be added in the fridge with 0 amount, with this the default amount of items is 1 if amount is not defined.